specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+        "target"
+      ]
+    },
+    "blobDetails": {
+      "description": "Blob details.",
+      "type": "object",
+      "properties": {
+        "containerName": {
+          "type": "string",
+          "description": "The container name."
+        },
+        "blobName": {
+          "type": "string",
+          "description": "The blob name."
+        }
+      },
+      "required": [
+        "containerName"
+      ]
+    },
+    "sasUriResponse": {
+      "description": "Get SAS URL operation response.",
+      "type": "object",
+      "properties": {
+        "sasUri": {
+          "type": "string",
+          "description": "A URL with a SAS token to upload a blob for execution in the given workspace."
+        }
+      }
+    },
+    "providerStatusList": {
+      "description": "Providers status.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/providerStatus"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Link to the next page of results."
+        }
+      }
+    },
+    "providerStatus": {
+      "description": "Providers status.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Provider id."
+        },
+        "currentAvailability": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Provider availability.",
+          "enum": [
+            "Available",
+            "Degraded",
+            "Unavailable"
+          ],
+          "x-ms-enum" : {
+            "name": "ProviderAvailability",
+            "modelAsString": true
+          }
+        },
+        "targets": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/targetStatus"
+          }
+        }
+      }
+    },
+    "targetStatus": {
+      "description": "Target status.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Target id."
+        },
+        "currentAvailability": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Target availability.",
+          "enum": [
+            "Available",
+            "Degraded",
+            "Unavailable"
+          ],
+          "x-ms-enum" : {
+            "name": "TargetAvailability",
+            "modelAsString": true
+          }
+        },
+        "averageQueueTime": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Average queue time in seconds."
+        },
+        "statusPage": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A page with detailed status of the provider."
+        }
+      }
+    },
+    "restError": {
```

Comment:
Your operations declares in "default" that this is the return type, hence this means errors would return something like:
```json
{
  "code": "code of the problem",
  "message": "message of the problem"
}
```

Microsoft open API guidelines recommendes to follow ODataV4:
```json
{
  "error": {
      "code": "code of the problem",
      "message": "message of the problem"
   }
}
```
http://docs.oasis-open.org/odata/odata-json-format/v4.0/os/odata-json-format-v4.0-os.html#_Toc372793091